.gitignore

```diff
@@ -15,6 +15,7 @@
 
 #pycache
 _pycache_
```

Comment:
There is a typo here, should have double underscores rather than single, thus it didn't work as expected.

Reply:
changed that